src/main/java/net/mcreator/preferences/data/HiddenSection.java

```diff
@@ -41,6 +38,8 @@ public class HiddenSection extends PreferencesSection {
 	public StringEntry uiTheme;
 	public BooleanEntry enableJavaPlugins;
 
+	public FileEntry lastWorkspace;
+
 	HiddenSection(String preferencesIdentifier) {

```

Comment:
java_home in this section should be FileEntry too from now on

Reply:
Just a question about `java_home`, where is it set to a value? I checked where it's used to see if it was a file or a folder that was set to the preference and the 3 places using the preference entry only get its value. It seems the only way to change its value is to write it inside the preferences file (in the `.mcreator folder`)
![image](https://user-images.githubusercontent.com/38427877/236708139-a8133736-6212-4b71-9d49-f1343aef7469.png)

Reply:
It is meant for advanced users to be able to override java_home in preferences file if needed